public-ui/src/app/utils/constants.tsx

```diff
@@ -14,6 +14,15 @@ export const surveyIdToRoute = {
     '43528698': 'family-health-history',
     '1333342': 'covid-19-participant-experience'
 };
+export const routeToSurveyId = {
```

Comment:
Nit: Will this be used in the future? Doesn't seem to be referenced anywhere currently.